htdocs/wp-content/themes/owc-formulieren/assets/js/editor/news/edit.js

```diff
@@ -0,0 +1,103 @@
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import {
+	useBlockProps,
+	InspectorControls,
+} from '@wordpress/block-editor';
+import {
+	TextControl,
+	Panel,
+	PanelBody,
+	Placeholder,
+} from '@wordpress/components';
+
+const Edit = ( props ) => {
+	const { attributes, setAttributes } = props;
+	const { apiUrl, amountOfPosts, detailPageUrl } = attributes;
+
+	const blockProps = useBlockProps( {
+		className: 'openpdd-news',
+		style: {
+			backgroundColor: '#fff',
+			padding: '1.5rem',
+			border: '1px solid rgba(0, 0, 0, 0.1)',
+		},
+	} );
+
+	const isValidUrl = urlString => {
+		const urlPattern = new RegExp( '^(https?:\\/\\/)?' + // validate protocol
+		                               '((([a-z\\d]([a-z\\d-]*[a-z\\d])*)\\.)+[a-z]{2,}|' + // validate domain name
+		                               '((\\d{1,3}\\.){3}\\d{1,3}))' + // validate OR ip (v4) address
+		                               '(\\:\\d+)?(\\/[-a-z\\d%_.~+]*)*' + // validate port and path
+		                               '(\\?[;&a-z\\d%_.~+=-]*)?' + // validate query string
+		                               '(\\#[-a-z\\d_]*)?$', 'i' ); // validate fragment locator
+		return !!urlPattern.test( urlString );
+	};
```

Comment:
Deze wat mag mij betreft in `assets/js/editor/util/helpers.js`. Dan is hij vaker inzetbaar en testbaar.